src/models/OAuthModel.php

```diff
@@ -125,20 +125,25 @@ public function createAccessTokenFromPassword($clientId, $username, $password)
      */
     protected function getUserId($username, $password)
     {
-        $sql  = 'SELECT ID, password, email FROM user
-            WHERE username=:username
-            AND verified = 1';
+        $sql  = 'SELECT ID, password, email, verified FROM user
+            WHERE username=:username';
         $stmt = $this->_db->prepare($sql);
         $stmt->execute(array("username" => $username));
```

Comment:
well, things can go wrong and ```execute``` can return ```false``` instead of true… 

But I'm perfectly fine with not doing it!